alpha/declcfg/load.go

```diff
@@ -196,7 +183,17 @@ func sendPaths(ctx context.Context, root fs.FS, pathChan chan<- string) error {
 	})
 }
 
-func parsePaths(ctx context.Context, root fs.FS, pathChan <-chan string, cfgChan chan<- *DeclarativeConfig) error {
+type pathMetas struct {
+	path  string
+	metas []pathMeta
+}
+
+type pathMeta struct {
+	meta *Meta
+	err  error
+}
```

Comment:
What are these for? I can't seem to see where they are being used, am I just missing it?

Reply:
Oh good catch. PR went through some optimization iteration and these are now unused.